.github/CONTRIBUTING.md

```diff
@@ -505,15 +505,14 @@ in the SQL/updates folder.
 * For map edit PRs, we do not accept 'change for the sake of change' remaps, unless you have very good reasoning to do so. Maintainers reserve the right to close your PR if we disagree with your reasoning.
 
 * Map Merge
-  * The following guideline for map merging applies to people who are **NOT** using StrongDMM, please see the StrongDMM section if you are.
-    * You **MUST** run Map Merge prior to opening your PR when updating existing maps to minimize the change differences (even when using third party mapping programs such as FastDMM.)
-    * Failure to run Map Merge on a map after using third party mapping programs (such as FastDMM) greatly increases the risk of the map's key dictionary becoming corrupted by future edits after running map merge. Resolving the corruption issue involves rebuilding the map's key dictionary;
+  * The following guideline for map merging applies to **aALL** mapping contributers.
```

Comment:
```suggestion
  * The following guideline for map merging applies to **ALL** mapping contributers.
```
The spelling obsessed man has made a spelling error. Laugh at this user.

Reply:
oh boy